tests/main/snap-run-hook/task.yaml

```diff
@@ -15,9 +15,6 @@ prepare: |
 restore: |
```

Comment:
I believe we don't need to remove those manually as `install_local` handles that now.

Reply:
Maybe I'm missing something, but I don't see any cleanup code in install_local() in snaps.sh. It would be very desirable though.

Reply:
Right, sorry. What I meant was that install_local caches the built snap at a separate location and that we don't have to remove the built snap explicitly.